fec-template/public/init.js

```diff
@@ -26,3 +27,39 @@ new feedback.Feedback();
 new typeahead.Typeahead('.js-search-input', $('.js-search-type').val());
 
 new typeaheadFilter.TypeaheadFilter('.js-typeahead-filter', typeahead.datasets.committees);
+
+$('.js-accordion').each(function(){
+  var contentPrefix = $(this).data('content-prefix') || 'accordion';
+  var openFirst = $(this).data('open-first') || false;
+  var opts = {
+    contentPrefix: contentPrefix,
+    openFirst: openFirst
+  };
+  new accordion.Accordion({}, opts);
```

Comment:
houndci has a point, it seems a bit weird to use `new` here. But it looks like [aria-accordian](https://github.com/stephenmathieson/aria-accordian/blob/master/index.js#L25) does handle the `new` for you.

Maybe we should disable this rule?